src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    name="Personal information"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label><br>
+
+        <label class="form-field" >
+          Name:
+          <input class="distance"
+          type="text"
+          name="name" autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            class="distance"
            type="text"
            name="name" autocomplete="off"
          >
```

*And the same for all such cases*